pkg/client/app.go

```diff
@@ -223,13 +223,24 @@ func translatePermissions(err error) error {
 func (c *DefaultClient) AppLog(ctx context.Context, name string, opts *LogOptions) (<-chan apiv1.LogMessage, error) {
 	appName := name
 
+	var targetNamespace, targetName string
+
 	app, err := c.AppGet(ctx, appName)
```

Comment:
This error isn't really being checked unless it is a not found error. Thoughts on if it should or shouldn't?

Reply:
I think we don't need to care too much about it.
If it's a real error, like not being able to contact the server, then subsequent requests like [ContainerReplicaGet](https://github.com/acorn-io/acorn/pull/1673/files/c5dd2cac8aa06789a287d83540e09327fdd87c8d#diff-2b548167cebba53ceca0681a297fd5471f930c043e4f6e50bda1eccaf78e89dcR236) will fail as well and bubble up the error. Other errors related to the input are handled properly by trying adjusted lookups.
Any objections?